sdk/cosmosdb/cosmos/CHANGELOG.md

```diff
@@ -1,9 +1,28 @@
 # Release History
 
-## 3.12.0 (Unreleased)
+## 3.12.0 (2021-07-06)
 
 ### Features Added
 - With the dropping of support for Node.js versions that are no longer in LTS, the dependency on `@types/node` has been updated to version 12. Read our [support policy](https://github.com/Azure/azure-sdk-for-js/blob/main/SUPPORT.md) for more details.
+- Added background refresher for endpoints, and new `ConnectionPolicy` options. Refreshing defaults to true, and the default refresh rate is every 5 minutes.
+```js
+const client = new CosmosClient({
+  endpoint,
+  key: masterKey,
+  connectionPolicy: {
+    ...defaultConnectionPolicy,
+    endpointRefreshRateInMs: 700,
+    enableBackgroundEndpointRefreshing: true
+  }
+})
+```
+
+- Added `client.dispose()` for closing the endpoint refresher verbosely. Necessary when destroying the CosmosClient inside existing processes like an express web server, or
+when you want to destroy the client and create a new one in the same process.
```

Comment:
is the line break between or and when needed?

Reply:
nope, fixed